src/test/java/org/jenkinsci/plugins/configfiles/ConfigFilesSEC1253Test.java

```diff
@@ -55,7 +56,8 @@ public void regularCaseStillWorking() throws Exception {
         assertThat(store.getConfigs(), hasSize(1));
 
         HtmlPage configFiles = wc.goTo("configfiles");
-        HtmlAnchor removeAnchor = configFiles.getDocumentElement().getFirstByXPath("//a[contains(@onclick, 'removeConfig?id=" + CONFIG_ID + "')]");
+        String attribute = j.jenkins.getVersion().isOlderThan(new VersionNumber("2.324")) ? "onclick" : "data-url";
```

Comment:
Here and elsewhere, we make the test compatible with both old and new versions of Jenkins by choosing the correct attribute based on the Jenkins version of the code under test.